Source/Cvent.SchemaToPoco.Core/JsonSchemaResolver.cs

```diff
@@ -234,25 +239,42 @@ private JsonSchemaWrapper ResolveSchemaHelper(Uri curr, Uri parent, string data)
         /// <returns>The JSON schema with standardized $ref attributes.</returns>
         private string StandardizeReferences(Uri parentUri, string data)
         {
-            var lines = new List<string>(data.Split('\n'));
-            var pattern = new Regex(@"(\""\$ref\""\s*:\s*\"")(.*)(\"")");
+            var lines = new List<string>(data.Split('\n'));
+            var idPattern = new Regex(@"(\""id\""\s*:\s*\"")(.*)(\"")");

```

Comment:
I don't think you need `\` in front of `""`; the double-quote is enough of an escape. The `.*`s here should probably be `.*?`. And I know these regexes won't work if there is an escaped quote somewhere in the JSON value, but that might be okay considering what these values are.